experiments/ClimaCore/bc-wave-slab/slab.jl

```diff
@@ -58,7 +58,7 @@ end
 
 function slab_init(
```

Comment:
Files modified in `experiments/ClimaCore/...` are unrelated to this PR and should be removed/reverted.